src/js/routes/nodes.js

```diff
@@ -45,12 +46,8 @@ let nodesRoutes = {
         {
           type: Route,
           name: 'nodes-task-panel',
-          path: 'task-detail/:taskID/?'
-        },
-        {
-          type: Route,
-          name: 'node-detail-tasks',
-          path: 'task-detail/:taskID/?'
+          path: 'task-detail/:taskID/?',
```

Comment:
Why do we even need this route again? Can't we just use the one in `services.js` for the task detail route?